build.gradle.kts

```diff
@@ -226,3 +228,14 @@ allprojects {
     environment("ZIPLINE_ROOT", rootDir.toString())
   }
 }
+
+subprojects {
+  plugins.withId("binary-compatibility-validator") {
+    configure<ApiValidationExtension> {
+      // Making this properly internal requires some SQLDelight work.
```

Comment:
Is there an existing SQLDelight issue for this that we can link?

Reply:
It’s work on our end, not theirs. SQLDelight visibility is always public, so we need to encapsulate through some other means.